code/datums/helper_datums/pconstruction_datum.dm

```diff
@@ -0,0 +1,260 @@
+#define FORWARD -1
+#define BACKWARD 1
+
+/*
+This only exists so I don't break mecha construction
+because the system mecha uses sucks horribly
+
+so i'm using paradise's superior construction datum
+
+which looks 100x better in practice and is just generally better
+*/
+
+/datum/pconstruction
```

Comment:
What exactly is wrong with ```/datum/construction```? from a cursory glance it is identical to ```/datum/pconstruction``` but without objectives (which we don't do/need anyway), and our ```/datum/construction``` actually matches our code standards.

The main issue is we can't have a second construction system, when we already have (partially complete) plans to overhaul our existing one with a FAR superior system to both